src/System.Runtime.InteropServices.RuntimeInformation/src/RuntimeInformation.cs

```diff
@@ -0,0 +1,47 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Diagnostics;
+
+namespace System.Runtime.InteropServices
+{
+    public static partial class RuntimeInformation
+    {
+        private const string RuntimeName = ".NET Core";
+        private static Architecture? s_processArch = null;
+        private static readonly bool s_is64BitProcess = IntPtr.Size == 8;
+
+        public static string FrameworkDescription
+        {
+            get
+            {
+                return RuntimeName;
```

Comment:
@Priya91 I suspect @davidfowl isn't asking for another property with the version. He is asking for some kind of marker, version or such, be added to the description string. I'm not sure what is the best marker but perhaps getting the file version of mscorlib or perhaps even this assembly would be a good enough marker to add to this string.